Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -147,14 +144,15 @@ void ABenchmarkGymGameModeBase::TryInitialiseExpectedActorCounts()
 
 void ABenchmarkGymGameModeBase::BuildExpectedActorCounts()
 {
-	AddExpectedActorCount(ExpectedNPCsCount, NPCClass, TotalNPCs, 1);
-	AddExpectedActorCount(ExpectedSimPlayersCount, SimulatedPawnClass, ExpectedPlayers, 1);
+	// Zoning scenarios can report actor count numbers slightly higher than the expected number so add a little slack.
```

Comment:
Why?

Reply:
Since the server report their actor counts at different times, when there is a transition of actors between zones, there will be some sync issues. In one case, the server gaining authority over the actor may report it's actor count before the server losing authority. In this case the calculated actor count would be high than expected. Will add a comment to summarize this.